tests/units/anta_tests/test_interfaces.py

```diff
@@ -1427,7 +1427,13 @@
             },
         ],
         "inputs": {"number": 2},
-        "expected": {"result": "failure", "messages": ["The following Loopbacks are not up: ['Loopback666']"]},
+        "expected": {
+            "result": "failure",
+            "messages": [
+                "Interface: Loopback666 - Invalid line protocol status - Expected: up - Actual: down",
+                "Interface: Loopback666 - Invalid interface status - Expected: connected -  Actual: notconnect",
```

Comment:
Extra space + same comment as above.
```suggestion
                "Interface: Loopback666 - Invalid interface status - Expected: connected Actual: notconnect",
```